tests/integration/test_workflow_tracer_vcr.py

```
@@ -0,0 +1,316 @@
+
+            # ---------------------------------------------------------------
+            # Phase 4: generate (Briefing Doc — fastest, text-only artifact)
+            # ---------------------------------------------------------------
+            gen = await client.artifacts.generate_report(
+                notebook_id, report_format=ReportFormat.BRIEFING_DOC
+            )
+            assert gen.task_id, "generate_report must return a task_id"
+            assert gen.status != "failed", f"generation failed: {gen.error}"
+
+            # Poll for completion. The report endpoint typically resolves
+            # within 30–60 s; cap at 5 minutes to leave headroom for slow
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)